### PR TITLE
Add-Deeps-Below

### PR DESCRIPTION
**Pull request by PlatypusKing**
Updated the following

monsters.js, 
maps.js, 
equipment.js 

Added information for the "Deeps Below" map.  This is a rough pass and will need testing/balance.still but should give atleast a base to work with.

### CODE REVIEW

File: src/game/models/monsters.js [hunk 1]
```diff
@@ -634,89 +634,88 @@ t . Dynamo
...
+      acc: 150,
+      con: 165,
+      flow: 150,
+      level: 215,
+      prizes: {
+        exp: 220,
+        items: ['Shattered blade'],
+        tokens: ['Pandoras Box'],
+ },
+,
```
Comment (KurzedMetal) [inline src/game/models/monsters.js]: Extra comma here, not needed

File: src/game/models/monsters.js [hunk 2]
```diff
@@ -634,89 +634,88 @@ t . Dynamo
...
+     ref: 130,
+     acc: 150,
+     con: 130,
+     flow: 150,
+     level: 200,
+     prizes: {
+       exp: 200,
+       items: ['Spider Leg'],
+       tokens: ['Pandoras Box'],
+    },
```
Comment (KurzedMetal) [inline src/game/models/monsters.js]: This curly bracket closes the prizes one, but it's not at the same indentation level (an extra space is needed)

File: src/game/models/monsters.js [hunk 3]
```diff
@@ -634,89 +634,88 @@ t . Dynamo
...
+},
```
Comment (KurzedMetal) [inline src/game/models/monsters.js]: The rest of the file is indenting with 2 spaces the first curly bracket in monsters definition.

Sticking to a single style is important for legibility

File: src/game/models/monsters.js [hunk 4]
```diff
@@ -634,89 +634,88 @@ t . Dynamo
...
+      id: 'urork',
```
Comment (KurzedMetal) [inline src/game/models/monsters.js]: This monster definition is giving a test error, it has no starting/opening curly bracket.

File: src/game/models/monsters.js [hunk 5]
```diff
@@ -634,89 +634,88 @@ t . Dynamo
...
+      con: 165,
+      flow: 150,
+      level: 215,
+      prizes: {
+        exp: 220,
+        items: ['Shattered blade'],
+        tokens: ['Pandoras Box'],
+ },
+,
+ {
```
Comment (KurzedMetal) [inline src/game/models/monsters.js]: Another bracket not using the proper indentation.

At this point I'm just pointing them out so you don't forget about them :)

**Comment by MarcoWorms** [comment]
Moved to #115 so I can add some final touches :)

**Pull request closed without merging**